web/src/people.ts

```diff
@@ -616,6 +616,10 @@ export function pm_with_user_ids(message: Message | MessageWithBooleans): number
         typeof message.display_recipient !== "string",
         "Private messages should have list of recipients",
     );
+    // TODO: Ideally display_recipient would be not optional in LocalMessage
+    // or MessageWithBooleans, ideally by refactoring the use of
+    // `build_display_recipient`, but that seemed complicated to type.
```

Comment:
Usually I'd file an issue for cleaning this up, but I think it might make sense to defer that until we have a new format for `display_recipient` in message objects in the API itself, at which point we'll naturally clean this up anyway.